addons/plugin_refresher/refresh_button.gd

```diff
@@ -174,15 +173,15 @@ func _search_dir_for_plugins(relative_base_folder: String = ""):
 	var dir := DirAccess.open(path)
 	
 	for subdir_name in dir.get_directories():
-		var relative_folder = relative_base_folder.path_join(subdir_name)
+		var relative_folder := relative_base_folder.path_join(subdir_name)
```

Comment:
Fixes error on L183: The argument 1 of the function "path_join()" requires the subtype "String" but the supertype "Variant" was provided.

Reply:
Also L191: The argument 1 of the function "_search_dir_for_plugins()" requires the subtype "String" but the supertype "Variant" was provided.